Bottom: 77334935921eeb9bf6211781066d1e5dbdf3da81
Top:    6e828c50857ae808503e6dfbb1927dc59c701c87
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-11 16:56:19 -0500

Refresh of vmbuilder-host

---
diff --git a/ansible-playbook/host_vars/buildbot-worker.novatech-llc.com/privileges.yml b/ansible-playbook/host_vars/buildbot-worker.novatech-llc.com/privileges.yml
new file mode 100644
index 00000000..fa188ba9
--- /dev/null
+++ b/ansible-playbook/host_vars/buildbot-worker.novatech-llc.com/privileges.yml
@@ -0,0 +1,3 @@
+---
+
+ansible_user: vagrant
